sort-selection/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Changelog
 All notable changes to the "sort-selection" extension will be documented in this file.
 
+## 2.0.2 - 2021-04-29
+- Added extension icon
+- Extension renamed
```

Comment:
Please update on L2 of this file also (and anywhere else sort-selection is found)

Reply:
Do I must change `sort-selection`  and `sortSelection` to `sortDelimitedList`?

Reply:
it would be `sort-delimited-list`